src/router/movies.ts

```diff
@@ -60,6 +64,24 @@ const deleteHandler = (req: Request, res: Response) => {
     }
 }
 
+const handleMutation = (req: Request, res: Response, next: NextFunction) => {
+    const { authorization } = req.headers;
+    if(!MUTATION_METHODS.includes(req.method)) {
+        const token = authorization?.split(' ')[1];
```

Comment:
This line repeat many time in this func, it could be optimised
```js
const { authorization } = req.headers;
const token = authorization?.split(' ')[1]
req['username'] = token && usersStorage.authenticateUser(token)
if (MUTATION_METHODS.includes(req.method) && !req['username']) {
 next(new Error('No token was found'));
 return
 }
 next()
 
```
Or something like that

Reply:
Will change it in the next homework